server/utils/pagination/pagination.ts

```diff
@@ -0,0 +1,95 @@
+import { Page } from '../../models/pagination'
+
+interface MojPaginationArgs {
+  classes?: string
+  items: (
+    | {
+        type: 'dots'
+      }
+    | {
+        type: 'pageNumber'
+        selected: boolean
+        text: number
+        href: string
+      }
+  )[]
+  previous?: {
+    text: string
+    href: string
+  }
+  next?: {
+    text: string
+    href: string
+  }
+  results?: {
+    from: number
+    to: number
+    count: number
+  }
+}
+
+export default class Pagination {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(private readonly page: Page<any>) {}
```

Comment:
can you get rid of the es-lint-disable by using `unknown`?

Reply:
yes, Thank you! I've changed to unknown.